security/login-error.php

```diff
@@ -0,0 +1,25 @@
+<?php
+namespace Automattic\VIP\Security;
+
+function use_ambiguous_login_error( $error ) {
+	global $errors;
+	$err_codes = $errors->get_error_codes();
+
+	$err_types = [
+		'invalid_username',
+		'invalid_email',
+		'incorrect_password',
+		'invalidcombo',
+	];
+
+	foreach ( $err_types as $err ) {
+		if ( in_array( $err, $err_codes, true ) ) {
+			$error = '<strong>ERROR</strong>: The username/e-mail address or password is incorrect. Please try again.';
```

Comment:
Why are you shouting? It's already strongly emphasised with HTML - it doesn't need allcaps as well 😄 Besides, your screenshot shows that WordPress doesn't use caps as well.

Also, existing nomenclature uses "email", not "e-mail" (again, as seen in screenshot).

```suggestion
			$error = '<strong>Error</strong>: The username/email address or password is incorrect. Please try again.';
```